admin_guide/enabling_EFK.adoc

```diff
@@ -0,0 +1,357 @@
+= Aggregating Logs with the EFK Stack
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+:prewrap!:
+
+toc::[]
+
+== Overview
+
+Application developers can aggregate and view logs for a range of services using
+the EFK stack, which is a modified version of the
+https://www.elastic.co/videos/introduction-to-the-elk-stack[ELK stack]. This
+topic shows you how to deploy the stack using a template.
+
+Three components make up the EFK logging stack:
+
+* Elasticsearch: an object store where all logs are fed.
+* Fluentd: gathers logs from nodes and feeds them to Elasticsearch.
+* Kibana: a web UI for Elasticsearch.
+
+== Preperation for Deploying the EFK Stack
+
+Deploying the EFK stack requires:
+
+* A 'logging-deployer' secret
+* A 'logging-deployer' service account with the secret and create privileges
+* Sufficient volumes for ElasticSearch cluster storage
+* Optionally, a key and certificate for the Kibana server
+
+The deployer generates the objects necessary for communication within the
+cluster and defines secrets and templates to implement aggregated logging.
+
+. Create a new project. Once implemented in a single project, the EFK stack will apply to every project within your OpenShift cluster:
++
+----
+$ oc new-project <project_name>
+----
+
+. Create a link:../dev_guide/secrets.html[secret] for the deployer. The secret can be configured for your
+instance, but specific for the deployer.
++
+A set of files can be supplied in the secret:
++
+[cols="2",options="header"]
+|===
+|File Name
+|Description
+
+|*_kibana.crt_*
+|A browser-facing certificate for the Kibana server.
+
+|*_kibana.key_*
+|A key to be used with the Kibana certificate.
+
+|*_kibana-ops.crt_*
+|A browser-facing certificate for the Ops Kibana server.
+
+|*_kibana-ops.key_*
+|A key to be used with the Ops Kibana certificate.
+
+|*_server-tls.json_*
+|JSON TLS options to override the Kibana server defaults. Refer to NodeJS docs for available options and the default options for an example.
+
+|*_ca.crt_*
+|A certificate for a CA that will be used to sign all certificates generated by the deployer.
+
+|*_ca.key_*
+|A matching CA key.
+|===
++
+You can supply the files when creating a new secret:
++
```

Comment:
I do think this is clear enough. The values are certainly not going to work if they're pasted into any commands.